### ISSUE TITLE
Modify state variable function “（def get_observations(））”

### ISSUE THREAD

**Issue by 1725917163** [NVIDIA-Omniverse/OmniIsaacGymEnvs#174]
obs = torch.cat(
        (
            # torso_position[:, 2].view(-1, 1),
            vel_loc,
            angvel_loc * angular_velocity_scale,
            normalize_angle(yaw).unsqueeze(-1),
            normalize_angle(roll).unsqueeze(-1),
            normalize_angle(angle_to_target).unsqueeze(-1),
            up_proj.unsqueeze(-1),
            heading_proj.unsqueeze(-1),
            dof_pos_scaled,
            dof_vel * dof_vel_scale,
            sensor_force_torques.reshape(num_envs, -1) * contact_force_scale,
            actions,
        ),
        dim=-1,
    )

When I comment out this line of code "# torso_position[:, 2].view(-1, 1)," I find that the running effect is very different. How can I solve this problem?

[Screencast from 07-25-2024 02:08:14 PM.webm](https://github.com/user-attachments/assets/09c3f574-8b95-415b-85ea-b7287d0fb0f3)
